File: truth_set/go/resource/variable-initialisation/calculateDiscount.go
package main

import "fmt"

func calculateDiscount(isMember bool) float64 {
	var discount float64

	if isMember {
		discount = 10 // 10% discount for members
	}

	finalPrice := 100 - discount
	return finalPrice
}

func main() {
	fmt.Println(calculateDiscount(true))
	fmt.Println(calculateDiscount(false))
}
\ No newline at end of file

File: truth_set/javascript/resource/variable-initialisation/calculateDiscount.js
function calculateDiscount(isMember) {
    var discount;
    if (isMember) {
        discount = 10; // 10% discount for members
    }
    var finalPrice = 100 - discount;
    return finalPrice;
}
console.log(calculateDiscount(true));
console.log(calculateDiscount(false));